packages/instant/src/util/error_reporter.ts

```diff
@@ -0,0 +1,68 @@
+import { logUtils } from '@0x/utils';
+
+import { ROLLBAR_CLIENT_TOKEN, ROLLBAR_ENVIRONMENT } from '../constants';
+
+// Import version of Rollbar designed for embedded components
+// See https://docs.rollbar.com/docs/using-rollbarjs-inside-an-embedded-component
+// tslint:disable-next-line:no-var-requires
+const Rollbar = require('rollbar/dist/rollbar.noconflict.umd');
+
+const shouldAllowRollbar = () => {
+    if (ROLLBAR_ENVIRONMENT === 'development') {
+        return process.env.ROLLBAR_FORCE_DEVELOPMENT_REPORT ? true : false;
+    }
+    return true;
+};
```

Comment:
I find the current function of `configs => rollbar state` a bit confusing. Can't there just be a `ROLLBAR_ENABLED` var as far as Instant is concerned?

Also `ROLLBAR_ENVIRONMENT` just means `ENVIRONMENT` right?

Reply:
Yes, renamed

Reply:
LMK if the env var logic still seems confusing to ya

Reply:
much better now!